pkg/debug/checkups/time.go

```diff
@@ -0,0 +1,103 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+type systemTime struct {
+	summary string
+	status  Status
+}
+
+func (st *systemTime) Name() string {
+	return "System Time"
+}
+
+func (st *systemTime) Run(_ context.Context, extraFh io.Writer) error {
+	var (
+		urls = []string{
+			"https://developers.google.com/time/",
+			"http://sha256timestamp.ws.symantec.com/",
+		}
+	)
+
+	for _, url := range urls {
+		serverTime, err := getTimeFromURL(url)
+		if err != nil {
+			fmt.Fprintf(extraFh, "error from url %s: %v\n", url, err)
+			continue
+		}
+
+		serverTime = serverTime.UTC()
+
+		fmt.Fprintf(extraFh, "got time from %s: %v\n", url, serverTime)
+
+		systemTime := time.Now().UTC()
+
+		fmt.Fprintf(extraFh, "system time: %v\n", systemTime)
+
+		diff := systemTime.Sub(serverTime)
+		if diff < 0 {
+			diff = -diff
+		}
+
+		maxDiff := 5 * time.Minute
+		if diff > maxDiff {
+			st.summary = fmt.Sprintf("system time off by more than %f minutes, delta = %f minutes", maxDiff.Minutes(), diff.Minutes())
+			st.status = Failing
+			return nil
+		}
+
+		st.summary = fmt.Sprintf("system time is within %f minutes, delta = %f minutes", maxDiff.Minutes(), diff.Minutes())
+		st.status = Passing
+		return nil
+	}
+
+	// if we made it here, we never got valid server time
+	st.summary = "could not get valid server time"
+	st.status = Failing
```

Comment:
```suggestion
	st.status = Erroring
```

erroring Indicates a failing in the checkup, vs failing which indicates a bad install

Reply:
done